mocks/styles.css

```diff
@@ -0,0 +1,134 @@
+@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+html {
+
+    overflow-y: hidden;
+    font-family: 'Raleway', sans-serif;
+
+}
+
+.login-body {
+    background: linear-gradient(-45deg, #471303, #52142c, #0d3646, #0c473a);
+    background-size: 400% 400%;
+    animation: gradient 15s ease infinite;
+    height: 100vh;
+}
+
+.register-body {
+    background: linear-gradient(-45deg, #424116, #194111, #0a2833, #630f0f);
+    background-size: 400% 400%;
+    animation: gradient 15s ease infinite;
+    height: 100vh;
+}
+
+.email-verification-body {
+    background: linear-gradient(-45deg, #381b12, #3b0f38, #02394d, #462506);
+    background-size: 400% 400%;
+    animation: gradient 15s ease infinite;
+    height: 100vh;
+}
```

Comment:
Try to make these in a separate file. This way, it shouldn't mess with the default styles.